lms/static/js/spec/verify_student/make_payment_step_view_spec.js

```diff
@@ -142,25 +142,26 @@ define([
             });
 
             it('view containing verification msg when verification deadline is set and user is active', function() {
-                var verificationDeadlineDateFormat = 'Aug 14, 2016 at 23:59 UTC';
+                var verificationDeadline = '2016-08-14 23:59:00+00:00';
```

Comment:
@yro is correct. We're now sending a real datetime object (as unicode) to the template for DateUtils to format, rather than a pre=formatted string. Hence the change in the mocked data.